src/controllers/institution-controller.js

```diff
@@ -0,0 +1,163 @@
+// import * as projectService from '../services/projects-service'
+// import * as projectDetailService from '../services/project-detail-service'
+import * as institutionService from '../services/institution-service.js'
+import { models } from '../models/init-models.js'
+import { Sequelize } from 'sequelize'
+import sequelizeConn from '../util/db.js'
+
+
+// get all projects from user ?
+
+// get specific project from user ?
+
+// get institutions
```

Comment:
All these comments do not provide more context about the code. They can be deleted.